package.json

```diff
@@ -2,15 +2,16 @@
   "name": "edx",
   "version": "0.1.0",
   "dependencies": {
+    "backbone": "~1.3.2",
+    "backbone.paginator": "^2.0.5",
```

Comment:
I would prefer that we don't declare this as a dependency, but instead use the version distributed inside the UI Toolkit's node_modules directory. I'm thinking of it an internal implementation detail of the UI Toolkit.